client/api_requests/populateDB.js

```diff
@@ -53,10 +53,20 @@ async function populateDB(app_method, constructor, endpoint) {
 	let promisesArr = [];
 	while (true) {
```

Comment:
you can consolidate:
```
while cursor != '' :
    do stuff
    cursor = 
```

Reply:
we actually cant do that unless we specify a do while loop.
first cursor is always ''.
we dont get first cursor untill first request so wont execute unless do...while loop.

Reply:
https://dev.to/andre347/how-to-use-a-do-while-loop-for-api-pagination-375b